app/mainwindow.cpp

```diff
@@ -927,6 +919,11 @@ void MainWindow::markdownConverterChanged()
     }
 }
 
+void MainWindow::editorFontChanged()
```

Comment:
Well, it is related to the pull request as well as to me not being able to read&write C++/Qt well :flushed:
I thought ```lastUsedStyle()``` is just a private method so I created a new slot at ```MainWindow```. But instead I should have connected the ```editorFontChanged``` signal directly to ```lastUsedStyle()```

I'll fix it

Unfortunately I'm a beginner at qt, c++ ...